featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -632,48 +715,91 @@ def test_diamond_entityset(diamond_es):
                                       time_last=datetime(2011, 4, 8),
                                       feature_set=feature_set)
     df = calculator.run(np.array([0, 1, 2]))
+    if isinstance(df, dd.DataFrame):
+        df = df.compute()
     assert (df['SUM(stores.transactions.amount)'] == [94, 261, 128]).all()
     assert (df['SUM(customers.transactions.amount)'] == [72, 411, 0]).all()
 
 
 def test_two_relationships_to_single_entity(games_es):
-    es = games_es
-    home_team, away_team = es.relationships
+    pd_es = games_es
```

Comment:
This test run with both pandas and Dask, so we should stick with `es` instead of `pd_es` to avoid confusion.